ceno_zkvm/src/e2e.rs

```diff
@@ -879,8 +881,7 @@ pub fn verify<E: ExtensionField, PCS: PolynomialCommitmentScheme<E> + serde::Ser
         transcript,
         zkvm_proof.has_halt(&verifier.vk),
     )?;
-    // print verification statistics like proof size and hash count
-    tracing::info!("e2e proof stat: {}", zkvm_proof);
```

Comment:
this was move to proof ready before verify, so even verify failed we still can see proof size data